contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -75,7 +75,7 @@ export GOARCH=amd64
 
 # Build runc
 cd $SRC/
-git clone https://github.com/opencontainers/runc --branch release-1.0
+git clone https://github.com/opencontainers/runc --branch release-1.1
```

Comment:
Wondering; should this use the version that's specified in `script/setup/runc-version` ? https://github.com/containerd/containerd/blob/aee92eb46ee23eae791fc6fa3626f0bfb41b2801/script/setup/runc-version#LL1C7-L1C7

Or perhaps even use the `install-runc` script as a whole? https://github.com/containerd/containerd/blob/main/script/setup/install-runc